practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>Please make it clearer that this is a task that needs completing (e.g. by putting "task" in the title of the cell). Same comment applies elsewhere when you have created tasks.</p><p> </p><p>Maybe also make them apply <code>lyr</code> to <code>x</code>, to make the task more substantive</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Resolved in next commit, thanks :) </p>

Reply:
<p>Please add the task title elsewhere as well, wherever you have left something for the reader to complete</p>

Reply:
<p>It now just says <code>#@task </code>here. This does not seem correct to me.</p><p> Should it not be <code>#@title task</code> ? </p>